war/src/main/less/pluginSetupWizard.less

```diff
@@ -75,6 +75,12 @@
         box-sizing: border-box;
         padding: 4.2em 0 4.8em 0;
     }
+    
+    .jenkins-version {
+        float: left;
```

Comment:
Needed? AFAIK, by the default is left-centered. Thus we can avoid a float layout for this.